forc-plugins/forc-client/src/op/call/mod.rs

```diff
@@ -190,47 +195,104 @@ async fn get_wallet(
     }
 }
 
+pub(crate) struct Abi {
+    program: ProgramABI,
+    unified: UnifiedProgramABI,
+    #[allow(dead_code)]
```

Comment:
If we are not using `type_lookup` let's remove it

Reply:
https://github.com/FuelLabs/sway/pull/7196/commits/c7afeac2cbd87d4197dec138199f1b127c75c6f4